init-containers/hamm-cost-updater/hamm-cost-updater/hamm-notification-subscriber.json.ctmpl

```diff
@@ -0,0 +1 @@
+{{with $environment := env "ENVIRONMENT"}}{{with $secret := vault (printf "secret/dsde/firecloud/%s/hamm-cost-updater/hamm-notification-subscriber.json" $environment)}}{{$secret.Data | toJSON}}{{end}}{{end}}
```

Comment:
I would also like @RLuckom to weigh in on this question.

In the firecloud-develop way of doing things there was a manifiest.rb file that was responsible for figuring out what to render but also copying files wholesale from vault as what this file is doing. I am fine with switching to a new model but I want to be sure we are not loosing any required functionality.

Reply:
I don't believe that we are. We _are_ moving things around--the render code in firecloud-develop has some fancy ways to pull files to render directly from github rather than finding them; now we're baking templates into  the init containers.

Reply:
how do we plan to deal with common files like newrelic or tcell configs?

Reply:
That's a good question. I'd advocate for putting them in dsp-k8s-deploy and having a copy mechanism analogous to firecloud-develop

Reply:
so for NR, ideally I think we should use [sbt-newrelic](https://github.com/saksdirect/sbt-newrelic). I think it has a few nice things compared to previous way of attaching java agent via java options, such as easier local testing, easier version upgrade. But this requires we package our service via [sbt-native-packager](https://github.com/sbt/sbt-native-packager), which hamm is using but I don't think other apps are packaged this way.